src/main/java/ru/hogwarts/school/repository/FacultyRepository.java

```diff
@@ -0,0 +1,13 @@
+package ru.hogwarts.school.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.hogwarts.school.model.Faculty;
+
+import java.util.List;
+
+public interface FacultyRepository extends JpaRepository<Faculty, Long> {
+
+    List<Faculty> findByColor(String color);
+
+    Faculty findByNameOrColorIgnoreCase(String name, String color);
```

Comment:
```suggestion
    Faculty findByNameIgnoreCaseOrColorIgnoreCase(String name, String color);
```
иначе только цвет будет без учёта регистра